### PR TITLE
DOCS-2309: Publishes Calico Open Source 3.28.2

### PR DESCRIPTION
**Pull request by ctauchen**
@coutinhop @danudey 
Still needs:
* operator version
* flannel version, if applicable
* autogenerated RN items

### CODE REVIEW

File: calico_versioned_docs/version-3.28/releases.json
Comment (coutinhop) [inline calico_versioned_docs/version-3.28/releases.json]:
```suggestion
      "version": "v1.34.5"
```

**Review by coutinhop — APPROVED** [review]

**Comment by ctauchen** [comment]
Just needs a date fix, then Lgtm pending clean build

**Pull request merged**
